src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,32 @@
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+interface LinkStatus {
+  isActive: boolean;
+}
+
+const getLinkClass = ({ isActive }: LinkStatus) =>
+  classNames('navbar-item', { 'has-background-grey-lighter': isActive });
+
+export const Navbar = () => {
+  return (
```

Comment:
```suggestion
export const Navbar = () => (
```

you can avoid return here